### PR TITLE
MVP auto provisioning

### PR DESCRIPTION
**Pull request by gorosz**

### CODE REVIEW

File: default.cfg [hunk 1]
```diff
@@ -17,6 +17,11 @@
 # Optional: the frequency of checking the status of push notification (seconds)
 ; rest_poll_interval=1
 
+[starling_auto_provision]
```
Comment (krajorama) [inline default.cfg:20]: Add comment that: Auto provisioning is turned on by setting both emal and phone attribute bellow.

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,8 @@
+2.1.0 2019-06-27
+
+Auto provision user, according to LDAP attributes
```
Comment (krajorama) [inline CHANGELOG:3]: add that the phone number should start with + <country code> ...

File: default.cfg [hunk 3]
```diff
@@ -17,6 +17,15 @@
...
+# Auto-provisioning is turned on only when both attributes are supplied
+# The ldap attribute name that holds the user's email address
+; email_attribute=mail
+# Tha ldap attribute name that holds the user's mobile phone number
+# Phone numbers should start with + signe followd by the country code followd by the number
```
Comment (tobal) [inline default.cfg]: typo: sign, followed by

**Review by tobal — APPROVED** [review]

**Review by tobal — APPROVED** [review]

**Review by krajorama — APPROVED** [review]
lgtm

**Pull request merged**